.github/workflows/bump-agent-versions.sh

```diff
@@ -22,7 +22,7 @@ else
     git add .agent-versions.json .package-version
 
     nl=$'\n' # otherwise the new line character is not recognized properly
-    commit_desc="These files are used for picking agent versions in integration tests.${nl}${nl}The content is based on responses from https://www.elastic.co/api/product_versions and https://snapshots.elastic.co${nl}${nl}The current update is generated based on the following requirements:${nl}${nl}Package version: ${package_version}${nl}${nl}\`\`\`json${nl}${version_requirements}${nl}\`\`\`"
+    commit_desc="These files are used for picking the starting (pre-upgrade) agent versions in upgrade integration tests.${nl}${nl}The content is based on responses from https://www.elastic.co/api/product_versions and https://snapshots.elastic.co${nl}${nl}The current update is generated based on the following requirements:${nl}${nl}Package version: ${package_version}${nl}${nl}\`\`\`json${nl}${version_requirements}${nl}\`\`\`"
```

Comment:
This is not entire true, the upgrade tests run both directions and these versions are not used in just upgrade tests.

Reply:
Thanks, I see now that the versions from this file are not just used as starting versions in the upgrade test.  For example, `TestStandaloneUpgradeRetryDownload` calls `upgradetest.PreviousMinor()` to set the end version and that function ends up looking at the versions from the file.  So I will fix the text in this PR accordingly.

But I'm not seeing where the versions from this file are being used outside of upgrade tests.  Could you point me to an example?

Reply:
@ycombinator yes, I meant the usage of `upgradetest.PreviousMinor()` in other tests, there is a few of them https://github.com/search?q=repo%3Aelastic%2Felastic-agent%20upgradetest.PreviousMinor()&type=code